### PR TITLE
Donation form on feedback message

### PR DESCRIPTION
**Pull request by symroe**

### CODE REVIEW

File: wcivf/apps/donations/helpers.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            interval_length=1,
+            interval_unit="month",
+            name=name,
+            description=description,
+            redirect_uri=settings.GOCARDLESS_REDIRECT_URL)
+
+
+
+
+# # client.new_subscription_url(
```
Comment (annapowellsmith) [inline wcivf/apps/donations/helpers.py]: might want to get rid of this

**Comment by symroe** [comment]
@annapowellsmith said:

>personally I would change "Help us do more..." to "Please can you help us do more...?", but you may disagree
> on negative page, I would add after "We're sorry you didn't find what you were looking for." something like "We'll read your feedback and pass it on to our volunteers."

@JoeMitchell do you have reckons?

**Comment by JoeMitchell** [comment]
Just looking at lots of other donate forms — it seems more assertive if you're ruder and don't say please :0  — would be one to A/B test!

I do like the second idea though — because it is true, I religiously read website-feedback and try to act on it where i can...

**Comment by symroe** [comment]
Updated:

screen shot 2017-05-31 at 18 19 43

**Comment by JoeMitchell** [comment]
Looks good!

**Pull request merged**